src/test/unit/pid_unittest.cc

```diff
@@ -381,39 +391,38 @@ TEST(pidControllerTest, testPidLevel)
     float currentPidSetpoint = 30;
     rollAndPitchTrims_t angleTrim = { { 0, 0 } };
 
-    currentPidSetpoint = pidLevel(FD_ROLL, pidProfile, &angleTrim, currentPidSetpoint, calcHorizonLevelStrength());
+    currentPidSetpoint = pidLevel(FD_ROLL, pidProfile, &angleTrim, currentPidSetpoint, calcHorizonLevelStrength(), true);
```

Comment:
I don't see the pidLevel function being invoked with the newRcFrame parameter as false anywhere in the code, so I assume this is not needed?